### PR TITLE
Fix text wrapping on program summary and preview view.

### LINKED ISSUE #1567: ORCA static question text is truncated
**Describe the bug**
Code of conduct text and link are truncated

**To Reproduce**
Click on apply to ORCA Middle School Opportunity Program
Scroll to bottom of ORCA application 

**Expected behavior**
Code of conduct text should be legible, but they are truncated on the right.

**Screenshots**
IMG_4011

**Smartphone (please complete the following information):**
 - Device: iPhone 8+
 - OS: iOS 14.7.1
 - Browser: safari
 - Version: latest

**Additional context**
N/A

### PR DESCRIPTION
**Pull request by ettengererin**
### Description
Bug addressed: Text does not wrap for long words, like copy/pasted URLs, in the application summary rows. We added custom CSS to break long words in order to wrap text. 

There is a similar tailwind.css property - https://tailwindcss.com/docs/word-break#break-words. However, the tailwind css does not actually behave as expected and does not solve the issue (looks like this property is not maintained by tailwind anymore).

Adding Bion as a reviewer here to confirm that adding custom css in this way is safe and to recommend any alternatives.

Before: 
image

After:
image

### Checklist
- [ ] Created tests which fail without the change (if possible)
- [ ] Extended the README / documentation, if necessary

### Issue(s)
Fixes #1567

### CODE REVIEW

**Comment by bion** [comment]
Looks like an important improvement!

Please use the tailwind style system whenever possible. In this case I believe you can add this constant to the class list to get the same result https://github.com/seattle-uat/civiform/blob/main/universal-application-tool-0.0.1/app/views/style/Styles.java#L2526

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java [hunk 1]
```diff
@@ -223,7 +223,8 @@ private ContainerTag renderQuestionSummary(
             Styles.P_2,
             Styles.PT_4,
             Styles.BORDER_B,
-            Styles.BORDER_GRAY_300);
+            Styles.BORDER_GRAY_300)
+        .attr("style", "word-break:break-word");
```
Comment (bion) [inline universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java:227]: Looks like an important improvement!

Please use the tailwind style system whenever possible. In this case I believe you can add this constant to the class list to get the same result https://github.com/seattle-uat/civiform/blob/main/universal-application-tool-0.0.1/app/views/style/Styles.java#L2526

**Review by bion — CHANGES_REQUESTED** [review]

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java (hunk 1 above)
Comment (bion) [reply universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java:227]: tailwind docs: https://tailwindcss.com/docs/word-break

Comment (ettengererin) [reply universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java:227]: We tried this tailwind property already, noted it in the PR description above:

Tailwind's break-words property does not actually behave as expected and does not solve the issue (looks like this property is not maintained by tailwind anymore). It does not properly wrap the long word text -- The URL link still overflows off the container. Seems like others experienced this problem: https://github.com/tailwindlabs/tailwindcss/issues/835

Break-words from tailwind adds this css:
{overflow-wrap: break-word;}

The custom css we use adds
{word-break: break-word}

Seems like older versions of tailwind worked properly for this use case, but the newer versions are causing this issue for folks.

Comment (bion) [reply universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java:227]: Ah! So sorry for not reading the description properly.

**Review by bion — APPROVED** [review]

File: universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java (hunk 1 above)
Comment (ettengererin) [reply universal-application-tool-0.0.1/app/views/applicant/ApplicantProgramSummaryView.java:227]: No worries!

**Pull request merged**